components/venue/VenueDivision.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+    <div class="venue-division overflow-hidden pt-6 md:pb-10 md:pt-16">
+        <div class="venue-division-container mx-[10px] flex-1">
+            <slot />
+        </div>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'VenueDivision',
+    props: {
+        index: { type: Number, default: -1 },
```

Comment:
**question:** what is `index` used for? I didn't see it referenced in this component.

Reply:
sorry, this should be removed!!

Reply:
fixed Thanks!